documentation/modules/proc-deploying-auth-services.adoc

```diff
@@ -26,19 +27,29 @@ openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=none-au
 ----
 {cmdcli} create secret tls none-authservice-cert --cert=none-authservice-cert/tls.crt --key=none-authservice-cert/tls.key
 ----
+endif::[]
 
 . Create the `none` authentication service:
 +
 [options="nowrap",subs="attributes"]
 ----
-{cmdcli} create -f ./resources/none-authservice/service.yaml
-{cmdcli} create -f ./resources/none-authservice/deployment.yaml
+{cmdcli} create -f ./examples/install/none-authservice
 ----
 
 == Deploying the `standard` authentication service
 
+.Prerequisites
+
+* Modify the installation files according to the namespace the Standard Authservice is going to be installed in
```

Comment:
Modify the installation files to reflect the namespace in which the Standard Authservice will be installed: